pkg/testsuite/testsuite.go

```diff
@@ -400,7 +400,12 @@ func (t *TestSuite) Run(failOnBlockFiltered bool, nodesOptions ...map[string][]o
 			}
 
 			if accountDetails.AccountID.Empty() {
-				accountDetails.AccountID = blake2b.Sum256(accountDetails.IssuerKey[:])
+				blockIssuerKeyEd25519, ok := accountDetails.IssuerKey.(iotago.BlockIssuerKeyEd25519)
```

Comment:
It doesn't need to be of type ed25519 I guess.
We could also directly hash the IssuerKeyBytes.